docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -0,0 +1,402 @@
+# Machine Learning Training
+
+Machine Learning Training allows to construct, control and train a machine learning model in Tizen devices.
```

Comment:
Machine Learning Training API/system/....?

Reply:
How about feature?

Reply:
this will get mixed with Tizen features ().
Further, you use API at bottom. Make sure whatever you choose works with it.

Reply:
Make it API, consistent wtih inference.

Reply:
Done